funfact/quantum/_special.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
How about merging this file with `lang._special`?

Reply:
Done